_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,135 @@
+---
+id: a73be2
+name: List elements follow content model
+rule_type: atomic
+description: |
+  This rule checks that list elements follow content model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility tree
+  - CSS Styling
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+The rule applies to every `ul` , `ol` , and `dl` HTML elements where the [semantic role][] is the same as the [implicit semantic role][].
+
+## Expectation
+
+Each test target has at least one [descendant][] in the [flat tree][] that is categorized as [palpable content][], which does not have the [hidden][] attribute specified.
```

Comment:
* `hidden` attribute is a very specific way of hiding stuff. Is it intentionally restricted to this way?
* What if the `hidden` attribute (or whatever) is not on the palpable content but on some ancestor? Failed Example 1 could have `hidden` on the `li`, or on a `div` surrounding the `p` (thus the `p` would be palpable without `hidden` attribute). I don't think this would be good.